src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -483,11 +446,7 @@ public enum TransmitFileOptions
     public partial class UdpClient : System.IDisposable
     {
         public UdpClient() { }
-        public UdpClient(int port) { }
-        public UdpClient(int port, System.Net.Sockets.AddressFamily family) { }
-        public UdpClient(System.Net.IPEndPoint localEP) { }
         public UdpClient(System.Net.Sockets.AddressFamily family) { }
-        public UdpClient(string hostname, int port) { }
```

Comment:
Ok, I thought we didn't want any of the constructors to do any connect-like operations, do we have any preference on the one that should be added back? What is the most used?